src/state/models/ui/preferences.ts

```diff
@@ -576,25 +558,25 @@ export class PreferencesModel {
     if (feedType === 'home' || feedType === 'following') {
       const feedTuners = []
 
-      if (areRepostsEnabled) {
-        feedTuners.push(FeedTuner.dedupReposts)
-      } else {
+      if (this.homeFeed.hideReposts) {
         feedTuners.push(FeedTuner.removeReposts)
+      } else {
+        feedTuners.push(FeedTuner.dedupReposts)
       }
 
-      if (areRepliesEnabled) {
+      if (this.homeFeed.hideReplies) {
+        feedTuners.push(FeedTuner.removeReplies)
+      } else {
         feedTuners.push(
           FeedTuner.thresholdRepliesOnly({
             userDid: this.rootStore.session.data?.did || '',
-            minLikes: repliesThreshold,
-            followedOnly: areRepliesByFollowedOnlyEnabled,
+            minLikes: Number(this.homeFeed.hideRepliesByLikeCount),
```

Comment:
This can be undefined right? Would result in NaN? I see there's an `|| 0` elsewhere.

Reply:
It can't actually ever be undefined due to JS serialization behaviors; the updated types PR will more properly reflect this.